templates/openstack/compare.html

```diff
@@ -37,10 +37,10 @@ <h1>OpenStack distributions: <br class="u-hide--small">a comparison</h1>
           <th>
             {{
               image(
-                url="https://assets.ubuntu.com/v1/5d6da5c4-logo-canonical-aubergine.svg",
+                url="https://assets.ubuntu.com/v1/563c0d9b-_Canonical.svg",
                 alt="",
```

Comment:
```suggestion
                alt="Canonical",
```

Reply:
I'm not sure how much more descriptive you could be, unless you want to add "logo" to it, but [from what I've read](https://www.a11yproject.com/posts/alt-text/) that isn't necessary:

"Make sure the description of the image is useful. For example, if the image is of a company's logo, the alt should be the company's name. The word "logo" is not necessary or useful as part of the alternative text"